libraries/AdaptiveExpressions/ExpressionFunctions.cs

```diff
@@ -2927,7 +2927,7 @@ private static IEnumerable<object> Flatten(IEnumerable<object> list, int dept)
                             var count = Convert.ToInt32(args[1]);
                             if (count <= 0)
                             {
-                                error = $"The second parameter should be more than zero";
+                                error = $"The second parameter ${args[1]} should be more than zero";
```

Comment:
>$ [](start = 63, length = 1)

I don't think this is right. did you mean to use LG syntax in the middle of C# code?

Reply:
fixed, also fix incorrect ${} in jpath and formatNumbers